src/styles/App.css

```
@@ -0,0 +1,16 @@
+html {
+  background-image: url("../assets/background.jpg");
+  background-size: cover;
```

To prevent you background from repeating when the viewport has different aspect ratios you could include
```suggestion
  background-size: cover;
  background-repeat: no-repeat;
  min-height: 100vh;
```